Tasks/ANT/anttask.ts

```diff
@@ -209,12 +209,13 @@ async function doWork() {
         // update JAVA_HOME if user selected specific JDK version or set path manually
         var javaHomeSelection = tl.getInput('javaHomeSelection', true);
         var specifiedJavaHome = null;
-
+        var javaTelemetryData = null;
```

Comment:
let is better than var (const is the best to use, but you can't here)

Reply:
I agree, I am sticking to var if the file has "vars" already. I realized these tasks need to be re-done in async/await pattern and that will be a good time to change the var to let.